tests/Testcontainers.ActiveMq.Tests/ActiveMqContainerTest.cs

```diff
@@ -0,0 +1,31 @@
+namespace Testcontainers.ActiveMq;
+
+public sealed class ActiveMqContainerTest : IAsyncLifetime
```

Comment:
Add scenarios for custom credentials and anonymous login, please. See [PR](https://github.com/testcontainers/testcontainers-java/pull/7400/files#diff-c40cca283c6efc68c1bbdfd2fe5399aeb56d4b8468c60be05560e164f5cc16b0) in Testcontainers for Java,